cinema/views.py

```diff
@@ -127,6 +125,25 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
When you return a response with a 400 status code, it's a good practice to also include a message in the response body explaining what went wrong to help the client understand the error.